lib/active_merchant/billing/gateways/spreedly_core.rb

```diff
@@ -200,6 +208,7 @@ def add_credit_card(doc, credit_card, options)
           doc.last_name(credit_card.last_name)
           doc.month(credit_card.month)
           doc.year(credit_card.year)
+          doc.verification_value(credit_card.verification_value) if credit_card.verification_value?
```

Comment:
@montdidier This value is already passed [here](https://github.com/activemerchant/active_merchant/pull/3102/files#diff-3d8ec2e8f34093ada56bc96cef17f9ecR206).

Did you intend to send the `verification_value ` only if it is present instead of always sending it?

Reply:
Good catch. It is just wrong.